src/Commands/InstallCommand.php

```diff
@@ -69,7 +70,19 @@ public function handle(Filesystem $filesystem)
         $this->call('hook:setup');
 
         $this->info('Publishing the Voyager assets, database, language, and config files');
-        $this->call('vendor:publish', ['--provider' => VoyagerServiceProvider::class]);
+
+        //Publish only relevant resources on install
+        $tags = ['voyager_assets', 'seeds', 'demo_content', 'config', 'migrations', 'lang'];
+
+        if (app()->version() >= 5.4) {
+            $tags = array_diff($tags, ['lang']);
```

Comment:
Same comments as here: https://github.com/the-control-group/voyager/pull/2343#pullrequestreview-82862437.  Also, having this here as well will cause merge conflicts for whichever PR gets merged last (this or #2343)